### PR TITLE
Update Dialogs Sample

### PR DESCRIPTION
**Pull request by DohyungLim**
- Change the text and name of the sample.
- Button and Controller list will be changed later.
- 3 Stacked button is not implemented.

### CODE REVIEW

File: examples/mobile/UIComponents/components/dialogs/dialog.html [hunk 1]
```diff
@@ -20,67 +20,67 @@ <h1>
...
 				</li>
 				<li class="ui-li-anchor">
-					<a data-position-to="window" data-rel="popup" href="#0-btn-title-popup">
-						0 Button Title Dialog
+					<a data-position-to="window" data-rel="popup" href="#no-action-dialog">
+						No Action Dialog
 					</a>
 				</li>
 				<li class="ui-li-anchor">
 					<a data-inline="true" data-position-to="window" data-rel="popup" href="#listview-popup">
```
Comment (HunseopJeong) [inline examples/mobile/UIComponents/components/dialogs/dialog.html]: How about changing the #listview-popup to the #listview-dialog to sync with title.

File: src/css/profile/mobile/common/popup.less [hunk 2]
```diff
@@ -57,6 +57,7 @@
 		overflow: hidden;
 		text-overflow: ellipsis;
 		white-space: nowrap;
+		font-family: Roboto-Medium;
```
Comment (HunseopJeong) [inline src/css/profile/mobile/common/popup.less]: Use the ".font(medium)"

File: examples/mobile/UIComponents/components/dialogs/dialog.html (hunk 1 above)
Comment (DohyungLim) [reply examples/mobile/UIComponents/components/dialogs/dialog.html]: I see. fixed.

done.

File: src/css/profile/mobile/common/popup.less (hunk 2 above)
Comment (DohyungLim) [reply src/css/profile/mobile/common/popup.less]: I modify your comment.

done.

File: src/css/profile/mobile/common/popup.less [hunk 3]
```diff
@@ -144,13 +149,7 @@
...
 
 	.ui-listview {
-		border-top-width: 1 * @px_base;
-		border-top-color: #e6e6e6;
-		border-top-style: solid;
-		border-bottom-width: 1 * @px_base;
-		border-bottom-color: #e6e6e6;
-		border-bottom-style: solid;
-		margin-left: -16 * @px_base;
+		margin-left: -23 * @px_base;
```
Comment (HunseopJeong) [inline src/css/profile/mobile/common/popup.less]: Is there any reason to set the margin left with - 23px?

Comment (DohyungLim) [reply src/css/profile/mobile/common/popup.less]: Dialog itself has padding on the left side. 
The listview inserted inside also has a margin on the left, so it looks skewed to the right.
so, to adjust the spacing, type a value of -23px.

Comment (HunseopJeong) [reply src/css/profile/mobile/common/popup.less]: Is the 24px of the dialog padding?
But why do you move to the left with -23px?

Comment (DohyungLim) [reply src/css/profile/mobile/common/popup.less]: When the header font was roboto-regular, 23px was an appropriate value.
But for roboto-medium, 24px is appropriate.
I will change it.

**Review by HunseopJeong — APPROVED** [review]
LGTM

**Pull request merged**